lib/i18n/hygiene/wrapper.rb

```diff
@@ -8,13 +8,18 @@ module Hygiene
     # queryable.
     class Wrapper
 
-      def initialize(keys_to_skip: [], locales: ::I18n.available_locales)
+      def initialize(keys_to_skip: [], scopes_to_exclude: [], locales: ::I18n.available_locales)
```

Comment:
Should we use consistent terminology across `keys_to_skip` and `scopes_to_exclude`? They are doing similar things.

I think I like exclude better for what it's worth.